archivebox/core/models.py

```diff
@@ -148,3 +151,18 @@ def save_tags(self, tags=[]):
             tags_id.append(Tag.objects.get_or_create(name=tag)[0].id)
         self.tags.clear()
         self.tags.add(*tags_id)
+
+
+class ArchiveResult(models.Model):
+    snapshot = models.ForeignKey(Snapshot, on_delete=models.CASCADE)
+    cmd = models.CharField(max_length=500, default="")
+    pwd = models.CharField(max_length=200, default="")
```

Comment:
Why allow emptystring here for `cmd` and `pwd`? Shouldn't these both always be set?

Reply:
I can remove them. I left it that way in case there were old indexes with those fields empty, but I think I could just ignore those entries if that case arises.